src/cdk/drag-drop/directives/drag.ts

```diff
@@ -254,6 +271,7 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
       .pipe(take(1), takeUntil(this._destroyed))
       .subscribe(() => {
         this._updateRootElement();
+        this._updatePreviewContainer();
```

Comment:
You can drop this call which will allow you to remove the `_updatePreviewContainer` method completely. The `_getPreviewContainer` call that you added in the `beforeStarted` subscription should be enough to handle all the cases since it fires right before dragging has started.

Reply:
done :-)